sonic-xcvrd/tests/test_xcvrd.py

```diff
@@ -219,99 +219,99 @@ def test_init_port_sfp_status_tbl(self):
+                                                                       'version_slot1_nic': '1.7',
+                                                                       'version_slot2_nic': '1.7',
+                                                                       'run_slot1_nic': 'True',
+                                                                       'run_slot2_nic': 'False',
+                                                                       'commit_slot1_nic': 'True',
+                                                                       'commit_slot2_nic': 'False',
+                                                                       'empty_slot1_nic': 'True',
+                                                                       'empty_slot2_nic': 'False',
+                                                                       'build_slot1_tor1': 'MS',
+                                                                       'build_slot2_tor1': 'MS',
+                                                                       'version_slot1_tor1': '1.7',
+                                                                       'version_slot2_tor1': '1.7',
+                                                                       'run_slot1_tor1': 'True',
+                                                                       'run_slot2_tor1': 'False',
+                                                                       'commit_slot1_tor1': 'True',
+                                                                       'commit_slot2_tor1': 'False',
+                                                                       'empty_slot1_tor1': 'True',
+                                                                       'empty_slot2_tor1': 'False',
+                                                                       'build_slot1_tor2': 'MS',
+                                                                       'build_slot2_tor2': 'MS',
+                                                                       'version_slot1_tor2': '1.7',
+                                                                       'version_slot2_tor2': '1.7',
+                                                                       'run_slot1_tor2': 'True',
+                                                                       'run_slot2_tor2': 'False',
+                                                                       'commit_slot1_tor2': 'True',
+                                                                       'commit_slot2_tor2': 'False',
+                                                                       'empty_slot1_tor2': 'True',
+                                                                       'empty_slot2_tor2': 'False'}))
     def test_post_port_mux_info_to_db(self):
         logical_port_name = "Ethernet0"
-        mux_tbl = Table("state_db", "mux_info_tbl")
+        mux_tbl = Table("STATE_DB", "MUX_CABLE_INFO")
```

Comment:
Suggest defining and using the following in y_cable_helper.py:

```python
MUX_CABLE_INFO_TABLE = "MUX_CABLE_INFO"
```

and referencing it here the other suggestions above (`xcvrd.y_cable_helper.MUX_CABLE_INFO_TABLE`)

Reply:
fixed